src/lib/core/gestures/gesture-config.ts

```diff
@@ -61,12 +70,25 @@ export class GestureConfig extends HammerGestureConfig {
    * @returns Newly-created HammerJS instance.
    */
   buildHammer(element: HTMLElement): HammerInstance {
-    const mc = new this._hammer(element, this._hammerOptions || undefined);
+    const hammer: HammerStatic = typeof window !== 'undefined' ? (window as any).Hammer : null;
+
+    if (!hammer) {
+      // If HammerJS is not loaded here, return the noop HammerInstance. This is necessary to
+      // ensure that omitting HammerJS completely will not cause any errors while *also* supporting
+      // the lazy-loading of HammerJS via the HAMMER_LOADER token introduced in Angular 6.1.
+      // Because we can't depend on HAMMER_LOADER's existance until 7.0, we have to always set
```

Comment:
Can you add an `@deletion-target` for 7.0 here so we remember to update it once 7.0 is out?